appeals/api/src/server/endpoints/appeals/appeals.formatter.js

```diff
@@ -230,7 +233,7 @@ export const mapAppealToDueDate = (appeal, appellantCaseStatus, appellantCaseDue
 			}
 			if (appeal.siteVisit) {
 				return addBusinessDays(
-					new Date(appeal.siteVisit.visitEndTime || appeal.siteVisit.visitDate),
+					new Date(appeal.siteVisit.visitEndTime || appeal.siteVisit.visitDate || 0),
```

Comment:
TS isn't keen on passing `null` to a `Date` constructor. `0` preserves the behaviour in the case that both `visitEndTime` and `visitDate` were null

Reply:
I mean 1/1/1970 is probably not the correct answer but I'm doing this PR to fix the types, not the code. The 0 preserves the behaviour and satisfies ts.

<img width="374" alt="Screenshot 2025-03-06 at 09 30 35" src="https://github.com/user-attachments/assets/72815491-64fb-47aa-a499-488039dcbdb8" />